specification/e-referrals-service-api.yaml

```diff
@@ -3650,22 +3650,68 @@ paths:
       security:
         - bearerAuth: []
       description: |
-        # TO ADD LATER
-      summary: A041 - Search service request
+        ## Overview
+
+        The e-RS represents an advice and guidance request and a referral in FHIR v4 as a ServiceRequest. This endpoint allows you to search for attributes about a ServiceRequest.
```

Comment:
I think we should say R4 rather than v4 as that is how FHIR refers to it.

Reply:
The wording has changed since I've added it. Will need to update.